index.js

```diff
@@ -109,7 +107,6 @@ function expressHasher(req) {
     return false;
   }
   // Examine the session
-  var safe = true;
```

Comment:
This variable was never used.

Reply:
Actually you found a serious bug here. The previous implementation of hasher was broken because it never returned `false` from the main function, the logic to check `safe` and return false if necessary was missing.

So we need to properly consult `safe`, not remove it.

Good catch.